### DISCUSSION TITLE
alltheplaces server down?

### DISCUSSION THREAD

**Discussion by Ebadm** [alltheplaces/alltheplaces#11325] [Q&A]
Can't seem to access these links:
https://www.alltheplaces.xyz/spiders
https://data.alltheplaces.xyz/runs/history.json

Are they working, if not, any reason why?

**Comment by riordan** [comment]
Now I feel like a jerk for opening an issue on this without checking here first 🤦‍♂️
- #11328

**Comment by iandees** [comment]
Sorry, didn't see the discussion post. This should be resolved! See the comment for background.
